src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -42,6 +41,18 @@ public static class PSVersionInfo
 
         private static readonly PSVersionHashTable s_psVersionTable;
 
+        /*
+            The following constants are generated by the source generator 'PSVersionInfoGenerator':
+
+                internal const string ProductVersion;
+                internal const string GitCommitId;
+
+                private const int Version_Major
+                private const int Version_Minor;
+                private const int Version_Patch;
+                private const string Version_Label;
```

Comment:
Are the names follow common naming standard?
Maybe MajorVersion and so on?

Reply:
What I intend to do is to make it look like `Version.Major`, `Version.Minor` and so on, to make their meaning more obvious.

As for naming standard, I did a quick search in PowerShell code base, and we do have const names using `_` in them, although most of them are const values for native code.

Personally, I think this is fine, especially that CodeFactor doesn't bark about it. But I'm willing to change if more people show concerns about the naming.

Reply:
It is minor. I am fine to merge as-is.